bootstrap/kubeadm/api/v1alpha3/suite_test.go

```diff
@@ -57,8 +57,8 @@ var _ = BeforeSuite(func(done Done) {
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths: []string{
-			filepath.Join("..", "..", "..", "..", "config", "bootstrap", "crd", "bases"),
-			filepath.Join("..", "..", "..", "..", "config", "controlplane", "crd", "bases"),
+			filepath.Join("..", "..", "config", "crd", "bases"),
+			filepath.Join("..", "..", "..", "..", "controlplane", "kubeadm", "config", "crd", "bases"),
```

Comment:
Does the bootstrapper actually need to load the controlplane CRDs for testing here? I would expect it to only need to load the bootstrapper and core CRDs.